srcpkgs/OpenCoarrays/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'OpenCoarrays'
+pkgname=OpenCoarrays
+version=2.9.2
+revision=1
+build_style=cmake
+hostmakedepends="gcc-fortran openmpi"
+makedepends="libgfortran-devel openmpi-devel"
+short_desc="Parallel application binary interface for Fortran 2018 compilers"
+maintainer="Alberto Ramos <alberto.ramos@ific.uv.es>"
+license="BSD-3-Clause"
+homepage="https://opencoarrays.org"
+distfiles="https://github.com/sourceryinstitute/OpenCoarrays/archive/${version}.tar.gz"
+checksum=a11639034ecc2445b6e5efa187ab4fac2b0b384ddae5872d62f717b40922d259
+nocross="openmpi not available on cross build"
```

Comment:
You don't need to specify this, the build system will detect that `openmpi` isn't available when building.